Sources/BottomController.swift

```diff
@@ -1,9 +1,23 @@
 import UIKit
 
 public protocol BottomControllerDelegate: class {
-     func requestToSwitchToTop(from bottomController: BottomController)
+     func requestToSwitchToTop(from bottomController: BottomControllable)
 }
 
-open class BottomController: UIViewController {
```

Comment:
Might be a good idea to rename the file as well from `BottomController.swift` to `BottomControllable.swift`